### PR TITLE
Closes [Term Entry] PHP String Functions chr() #3220

### PR DESCRIPTION
**Pull request by alinasir85**
### Description

- Added chr() term entry for php
- Closes issue #3220

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/chr/chr.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+The **`chr()`** function in PHP is used to convert an ASCII value into a single-character string.
+
+## Syntax
+
+```pseudo
+chr($ascii_value)
+```
+
+It accepts an ASCII value as a parameter and returns a string representing a character from the specified ASCII value.
```
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/chr/chr.md:23]:
```suggestion
It accepts an ASCII value as the argument and returns a string representing the associated character.
```

File: content/php/concepts/string-functions/terms/chr/chr.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+The ASCII value can be specified in decimal, octal, or hex values.
+
+- Octal values are defined by a leading 0.
+- Hex values are defined by a leading 0x.
```
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/chr/chr.md:28]:
```suggestion
- Octal values are defined by a leading `0`.
- Hex values are defined by a leading `0x`.
```

File: content/php/concepts/string-functions/terms/chr/chr.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+## Example
+
+The example below demonstrate the `chr()` function when different ASCII are passed but their equivalent character is same:
```
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/chr/chr.md:32]:
```suggestion
The example below demonstrate the `chr()` function when different ASCII values are passed but their equivalent character is same:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@alinasir85 I've added several comments. Please update the entry, and tag me when you've made the changes. Thanks

**Comment by alinasir85** [comment]
> @alinasir85 I've added several comments. Please update the entry, and tag me when you've made the changes. Thanks

@caupolicandiaz, suggested changes pushed

**Review by caupolicandiaz — APPROVED** [review]
@alinasir85 This is ready for a second review

**Review by SSwiniarski — APPROVED** [review]
@alinasir85 @caupolicandiaz Looks good.

**Pull request merged**